README.md

```diff
@@ -251,4 +263,77 @@ Not all steps need to be followed on the instructions for the aad-pod-identity p
     secret1
     ```
 
+#### OPTION 3: VMSS User Assigned Managed Identity
```

Comment:
Can we make this bold instead of a new header to be consistent with the rest of the readme?
`** OPTION 3 - VMSS User Assigned Managed Identity **`